test-e2e/basic.test.ts

```diff
@@ -23,13 +26,14 @@ test('basic', async ({ launch }) => {
   await page.getByRole('button', { name: 'Run Tests' }).click()
 
   // check results
-  await page.locator(`[title*="pass.test.ts (Passed)"]`).click()
-  await page.locator(`[title*="fail.test.ts (Failed)"]`).click()
-  await page.locator(`[title*="mix.test.ts (Failed)"]`).click()
-  await page.locator(`[title*="3/7 tests passed"]`).click()
+  await expect(page.locator(`[title*="pass.test.ts (Passed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="fail.test.ts (Failed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="mix.test.ts (Failed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="3/7 tests passed"]`)).toBeVisible()
 })
 
 test('imba', async ({ launch }) => {
+  await execa('npm', ['i'], { cwd: './samples/imba' })
```

Comment:
Does this fix https://github.com/vitest-dev/vitest-ecosystem-ci/actions/runs/8354143640/job/22867013035 ?

Reply:
Yeah, I think that's the same issue.

Reply:
Hmm, technically if `samples/imba` installed its own Vitest, then it's not testing against ecosystem-ci's Vitest.

Reply:
Superseded by
- https://github.com/vitest-dev/vscode/pull/311

I'll also setup a wrapper script for ecosystem-ci, so we can tweak things while keeping the same script on ecosystem-ci repo.